micro-benchmarks/DRB197-diffusion2_yes.c

```diff
@@ -0,0 +1,46 @@
+/*
+!!!~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~!!!
+!!! Copyright (c) 2017-20, Lawrence Livermore National Security, LLC
+!!! and DataRaceBench project contributors. See the DataRaceBench/COPYRIGHT file for details.
+!!!
+!!! SPDX-License-Identifier: (BSD-3-Clause)
+!!!~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~!!!
+ */
+
+/*
+ * This is a program based on a dataset contributed by
+ * Wenhao Wu and Stephen F. Siegel @Univ. of Delaware.
+
+ * Race due because u1 and u2 become aliased
```

Comment:
There is no variable named u1 or u2 in the code below. Do you mean u[0] and u[1]?